lib/gigya_flutter_plugin.dart

```diff
@@ -50,6 +51,11 @@ class GigyaSdk {
     return GigyaFlutterPluginPlatform.instance.webAuthenticationService;
   }
 
+  /// Get the biometric service provided by the Gigya SDK.
+  BiometricService get biometricService {
```

Comment:
Could you `export` the BiometricService interface? So that people can use its type. IIRC we also export the other service interfaces.

See https://github.com/SAP/gigya-flutter-plugin/blob/main/lib/gigya_flutter_plugin.dart#L29-L31